cparser/Elab.ml

```diff
@@ -1374,7 +1405,7 @@ let elab_expr vararg loc env a =
         match (unroll env b1.etyp, unroll env b2.etyp) with
         | (TPtr(t, _) | TArray(t, _, _)), (TInt _ | TEnum _) -> t
         | (TInt _ | TEnum _), (TPtr(t, _) | TArray(t, _, _)) -> t
-        | t1, t2 -> error "incorrect types for array subscripting" in
+        | t1, t2 -> error "subscripted value is not an array, or pointer" in
```

Comment:
"not array or pointer" (no comma).  Or, better English: "neither array nor pointer".  (The latter is GCC's wording.)